kfs-core/src/main/java/edu/arizona/kfs/fp/businessobject/lookup/UaDisbursementPayeeLookupableHelperServiceImpl.java

```diff
@@ -36,17 +41,18 @@
 	/**
 	 * @see org.kuali.kfs.fp.businessobject.lookup.DisbursementPayeeLookupableHelperServiceImpl#getPersonAsPayees(java.util.Map)
 	 */
+	@SuppressWarnings("deprecation")
 	protected List<DisbursementPayee> getPersonAsPayees(Map<String, String> fieldValues) {
 		List<DisbursementPayee> payeeList = new ArrayList<DisbursementPayee>();
 
-		List<String> employeeStatusCodes = (List<String>) CoreFrameworkServiceLocator.getParameterService().getParameterValuesAsString(DisbursementVoucherDocument.class, UaKFSConstants.ACTIVE_EMPLOYEE_STATUS_CODES_PARAM_NAME);
+		ArrayList<String> employeeStatusCodes = new ArrayList<String>(CoreFrameworkServiceLocator.getParameterService().getParameterValuesAsString(DisbursementVoucherDocument.class, UaKFSConstants.ACTIVE_EMPLOYEE_STATUS_CODES_PARAM_NAME));
 		if (ObjectUtils.isNull(employeeStatusCodes) || employeeStatusCodes.isEmpty()) {
-			return super.getPersonAsPayees(fieldValues);
+			employeeStatusCodes = new ArrayList<String>();

```

Comment:
This was the default status value used in super.getPersonAsPayees method, which is why I included it here. I can try changing this to throw an exception, but that would change the method signature, and I might have to make significant other changes to avoid breaking the server during runtime. I'll have to look into that.